Profiler/install.cmd

```diff
@@ -0,0 +1,79 @@
+@echo off
+setlocal EnableDelayedExpansion
+
+rem check for admin rights
+net session >nul 2>&1
+if %errorLevel% NEQ 0 (
+	echo Failure: Execute script as Administrator.
+	pause
+	exit /B 1
+)
+
+
+set InstallSource=%CD%
+for /f "delims=" %%x in (%InstallSource%\install.cfg) do (
+	set line=%%x
+	if "!line:~0,1!" neq "#" (
+		set !line!
+	)
+)
+
+if "%InstallDir%"=="" (
+	echo Failure: InstallDir not set.
+	pause
+	exit /B 1
+)
+
+echo Installing Profiler to: "%InstallDir%"
+
+if exist "%InstallDir%" (
+	if "%InstallUploadService%"=="true" (
+		echo Uninstalling Upload Service
+		pushd "%InstallDir%\UploadDaemon\service"
+		UploadDaemonService stop
+		UploadDaemonService uninstall
+		popd
+	)
+) else (
+	mkdir "%InstallDir%"
+)
+
+xcopy "%InstallSource%" "%InstallDir%" /e /y /q
+
+if exist "%PROGRAMFILES(X86)%" (
+	copy "%InstallDir%\Profiler64.dll" "%SystemRoot%\System32\TeamscaleProfiler.dll" /y
+	copy "%InstallDir%\Profiler32.dll" "%SystemRoot%\SysWOW64\TeamscaleProfiler.dll" /y
+) else (
+	copy "%InstallDir%\Profiler32.dll" "%SystemRoot%\System32\TeamscaleProfiler.dll" /y
+)
+
+setx /m COR_PROFILER {DD0A1BB6-11CE-11DD-8EE8-3F9E55D89593}
+
+REM Enable the profiler globally (the actual profiling is disabled in the Profiler.yml)
```

Comment:
Please ensure the user has actually created a yml file. By default, this doesn't exist and I don't know what happens then. Possibly, all .NET processes are being profiled, which is probably not intended.

Reply:
alternatively, ship with a "disable all" default config, though that might be confusing as we also ship with an example config.